ssh_check/datadog_checks/ssh_check/ssh_check.py

```diff
@@ -52,6 +53,17 @@ def _load_conf(self, instance):
             params.append(value)
         return self.Config._make(params)
 
+    def collect_version(self, version):
+
+        version_info = version.read().decode("utf-8")
+
+        try:
+            version = re.search(r'(\d+\.\d+.*),', version_info).group(1)
+            version_semver = version.replace('p', '.')
```

Comment:
It seems that ssh doesn't use semver, about the `p`:

> The portable OpenSSH follows development of the official version, but releases are not synchronized. Portable releases are marked with a 'p' (e.g. 4.0p1). The official OpenBSD source will never use the 'p' suffix, but will instead increment the version number when they hit 'stable spots' in their development.

Reply:
Since it's not `semver` you might want to use a custom version pattern like what's done here: https://github.com/DataDog/integrations-core/pull/4979/files